### PR TITLE
Modify my slide

### PR DESCRIPTION
**Pull request by waelelkadim**
# Demo for Pull Requests

- Add xyz
- update my slide

- [ ] Task 1
- [ ] Task 2

This PR Resolves #8

### CODE REVIEW

File: _slides/05-waelelkadim.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello from Wael Elkadim
```
Comment (elkadeem) [inline _slides/05-waelelkadim.md:6]: This line should be from 10 to 15 words.

**Review by elkadeem — COMMENTED** [review]
Fix the issues related to changes.

**Review by elkadeem — COMMENTED** [review]
Fix the issues related to changes.

**Review by elkadeem — APPROVED** [review]
Well done.

**Comment by atheer174** [comment]
well cone

**Review by abdulraoof1997 — APPROVED** [review]
Reviewed and good to go from my end

**Review by Khalid4447 — COMMENTED** [review]
perfect

**Review by Sa4d — COMMENTED** [review]
@waelelkadim  well done

**Review by Sa4d — COMMENTED** [review]
@waelelkadim  well done

**Pull request merged**